Minor fix re get-instance-view

jtuliani opened the pull request:
Fix: get instance-view changed to get-instance-view (to match the example immediate after)

azurecla commented:
Hi __@neilpeterson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nepeters). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Minor fix re get-instance-view

Fix: get instance-view changed to get-instance-view (to match the example immediate after)
